src/containers/HomePage.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
 import { connect } from 'react-redux';
-import { func } from 'prop-types';
+import { string } from 'prop-types';
 
+import Map from '../components/common/Map';
 import Menu from '../components/common/Menu';
-import { logout } from '../actions/sessionActions';
 
-const HomePage = ({ logout }) => (
+const HomePage = ({ user }) => (
   <div className="home-page">
-    <Menu />
-    <button onClick={logout}>
-      <FormattedMessage id="logout.button" />
-    </button>
+    <Menu user={user} />
```

Comment:
The `user` prop is just the user name. Please rename it to depict what it actually stores.